presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/StreamPropertyDerivations.java

```diff
@@ -362,6 +363,14 @@ public StreamProperties visitAggregation(AggregationNode node, List<StreamProper
             return properties.translate(symbol -> node.getGroupingKeys().contains(symbol) ? Optional.of(symbol) : Optional.empty());
         }
 
+        @Override
+        public StreamProperties visitAnalyzeFinish(AnalyzeFinishNode node, List<StreamProperties> inputProperties)
+        {
+            StreamProperties properties = Iterables.getOnlyElement(inputProperties);
+            // analyze finish only outputs TRUE
```

Comment:
outputs row count?

Reply:
Nice catch. I missed that we adding output to `ANALYZE`.